File: 프로그래머스/0/120813. 짝수는 싫어요/짝수는 싫어요.cs
using System;
using System.Collections.Generic;

public class Solution 
{
    public int[] solution(int n) 
    {
        List<int> oddList = new List<int>();
        
        for(int i=1; i<=n; i++)
        {
            if(i%2 == 1)
            {
                oddList.Add(i);  
            }
        }
        return oddList.ToArray();
    }
}

File: 프로그래머스/0/120813. 짝수는 싫어요/짝수는 싫어요.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(int n) 
{
    int i = 0;
    vector<int> vec;
    while(i<=n)
    {
        if(i%2 == 1)
        {
            vec.push_back(i);
        }
        i++;
    }
    return vec;
}